l3backend/l3backend-opacity.dtx

```diff
@@ -283,7 +283,7 @@
 \cs_new_protected:Npn \@@_backend_fill:n #1
   { \@@_backend:nn {#1} { fill- } }
 \cs_new_protected:Npn \@@_backend_stroke:n #1
-  { \@@_backend:nn { {#1} } { stroke- } }
+  { \@@_backend:nn {#1} { stroke- } }
```

Comment:
Do we need a ChangeLog entry for this?

Reply:
Added with https://github.com/latex3/latex3/pull/1479/commits/bedaf77333605f3b6dfca199676b2f498439256d.